src/screens/Sidebar/ReportBugs.js

```diff
@@ -0,0 +1,69 @@
+import React from 'react';
+import { View, WebView, ActivityIndicator } from 'react-native';
+import { connect } from 'react-redux';
+import { cannyBoardTokens } from 'kitsu/constants/app';
```

Comment:
I think it might make sense to move these to `config/env`? Also I gave you the wrong keys before 😞  the database requests one is right tho.
```
fed94acf-1df6-dda7-fdd0-7c86cb41eb63 mobile features
b3f2fed5-ca07-bf13-dbbb-6e9fea351064 mobile bugs
```